packages/visitor-keys/src/visitor-keys.ts

```diff
@@ -16,6 +16,9 @@ type AdditionalKeys = {
 };
 
 const additionalKeys: AdditionalKeys = {
+  // Stage 3 Import Assertions
+  ImportAttribute: [],
```

Comment:
You will need to declare this so that ESLint will traverse these nodes

```suggestion
  ImportAttribute: ['key', 'value'],
```